src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -88,14 +91,14 @@ static Command parse(String input) throws WrongFormatException, NumberFormatExce
             if(industry.equals(" ")){
                 ui.emptyInputErrorMessage("industry");
                 throw new WrongFormatException();
-            }else if(!checkIndustryValidity(industry)){
+            }else if(!isIndustryValid(industry)){
                 ui.invalidInputFormatErrorMessage("industry",
                         "Industry cannot be a word without alphabet.");
                 throw new WrongFormatException();
             }
             //Only valid Singaporean number is allowed.
             int contactNumber = Integer.parseInt(contactNumberString);
-            if(contactNumberString.length() > 8 || !checkContactNumberValidity(contactNumber)){
+            if(contactNumberString.length() > 8 || !isContactNumberValid(contactNumber)){
```

Comment:
Good change of method name to reflect that a Boolean is returned, perhaps include a space between `if` and the opening parenthesis as well as the closing parentheses and the opening curly braces to fix the minor coding standard violations.